flowtools/flowclr_sample_name/sample_simp.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="flclr_sample_name" name="Add sample information" version="1.0+galaxy1", profile="18.01">
+  <description>to flow clustered text file collection.</description>
+  <requirements>
+    <requirement type="package" version="1.42.0">bioconductor-flowcore</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="2:" level="fatal" description="See stderr for more details." />
+  </stdio>
+  <command><![CDATA[
+    Rscript '$__tool_directory__/sample_simp.R' '${input}' '${output}' '${samp_names}'  
```

Comment:
> (a) decompose the dataset collection as a concatenation of paths (possibly with commas) 

Needs to happen in the command block, before the R call, with a cheetah for loop over the elements of the dataset collection. See guidance [here](https://planemo.readthedocs.io/en/latest/writing_advanced.html#processing-lists-reductions).